### ISSUE TITLE
about the computer network experiments

### ISSUE THREAD

**Issue by AI678** [holazzer/qiang-networking#2]
Hey , I am a undergraduate student in China. And I really want to ask  does this course or tutorial have experiments for proctol and implemention respectively . I want to learn more about this. Thank you for your response.

**Comment by yhznb** [comment]
I have the same question. I also wonder whether this respository will update in the future. Thank you for your response. I will star you

**Comment by holazzer** [comment]
Yes, I will invite my dear friend whose name happens to also be yhz and he will design the experiments for you. He has multiple **HIGH CITED** works ! **high cited !** That's what he said.

**Comment by yhznb** [comment]
thank you very much. I think this repository maybe useless for Qiang and me.

**Comment by yhznb** [comment]
I think experiments are useful.
